vue/sbc-common-components/src/mixins/navigation-mixin.ts

```diff
@@ -1,13 +1,23 @@
 import { Component, Vue } from 'vue-property-decorator'
+import ConfigHelper from '../util/config-helper'
 
 @Component({})
 export default class NavigationMixin extends Vue {
   protected navigateTo (contextPath: string, routePath: string): void {
     const resolvedRoutes = this.$router.resolve({ path: `/${routePath}` })
+    console.log(resolvedRoutes.resolved.matched)
     if (resolvedRoutes.resolved.matched.length > 0) {
       this.$router.push(`/${routePath}`)
     } else {
       window.location.assign(`${contextPath}${routePath}`)
     }
   }
+
+  protected redirectToPath (inAuth: boolean, routePath: string) {
```

Comment:
navigateTo already handles route push if user is in auth page, else using window location. Do we still need this change here ?

Reply:
You are right, will keep it as a single func